app/api/migrations/migrations/25-sanitize-string-geolocations/index.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-await-in-loop */
+
+import logger from 'api/migrations/logger';
+
+const migrationName = 'sanitize-string-geolocations';
+
+function sanitizeGeolocation(property) {
+  if (!Array.isArray(property) || !property.length) {
+    return property;
+  }
+
+  const value = { ...property[0].value };
+
+  // eslint-disable-next-line no-new-wrappers
+  const lat = new Number(value.lat).valueOf();
+  // eslint-disable-next-line no-new-wrappers
+  const lon = new Number(value.lon).valueOf();
```

Comment:
@fnocetti there should be no need of eslint disable here, it should work if you remove the "new" operator i think ?